crates/bevy_render/src/lib.rs

```diff
@@ -582,7 +565,19 @@ unsafe fn initialize_render_app(app: &mut App) {
             ),
         );
 
-    render_app.set_extract(|main_world, render_world| {
+    // We want the closure to have a flag to only run the RenderStartup schedule once, but the only
+    // way to have the closure store this flag is by capturing it. This variable is otherwise
+    // unused.
+    let mut run_startup = true;
```

Comment:
The variable naming is ambiguous here. Does this mean "should run startup" or "has run startup". I'd prefer a rename to `should_run_startup`.